### DISCUSSION TITLE
Can't bind to 'item' since it isn't a known property of 'li'

### DISCUSSION THREAD

**Discussion by chungTA0609** [primefaces/community#892] [PrimeNG] [answered]
Hello, I recently purchased the "mirage-ng" theme, I'm having trouble configuring the menu:
CODE: 
```
<ng-container *ngFor="let item of model; let i = index">
    <li
      app-menu-item
      *ngIf="!item.separator"
      [item]="item"
      [index]="i"
      [root]="true"
    ></li>
    <li *ngIf="item.separator" class="menu-separator"></li>
  </ng-container>
```

is returning the following error:

Can't bind to 'item' since it isn't a known property of 'li'.ngtsc(-998002)
menu.component.ts(2, 79): Error occurs in the template of component MenuComponent.

Angular 17

**Comment by mehmetcetin01140** [comment, marked as answer]
Could you try using 'app-menuitem' instead of 'app-menu-item'?
